apps/web/assets/js/components/List.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import Opportunity from './Opportunity'
+import ReactPaginate from 'react-paginate'
+
+export default class List extends React.Component {
+   constructor(props){
+    super(props)
+
+    this.state = { entries: [] }
+    this.$opportunities(props, this.state.page)
+
+    this.handlePageClick = this.handlePageClick.bind(this)
+  } 
+
+  componentWillReceiveProps(nextProps) {
+    this.$opportunities(nextProps)
```

Comment:
I noticed this results in the page being undefined, changing it to the following fixes it. `this.$opportunites(nextProps, this.state.page)`

Reply:
Good catch @dberget!  I will test again locally and make the change 👍